zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/metrics/UsageMetricRecord.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.protocol.impl.record.value.metrics;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.camunda.zeebe.msgpack.property.DocumentProperty;
+import io.camunda.zeebe.msgpack.property.EnumProperty;
+import io.camunda.zeebe.msgpack.property.LongProperty;
+import io.camunda.zeebe.protocol.impl.encoding.MsgPackConverter;
+import io.camunda.zeebe.protocol.impl.record.UnifiedRecordValue;
+import io.camunda.zeebe.protocol.record.value.UsageMetricRecordValue;
+import java.util.Map;
+import org.agrona.DirectBuffer;
+
+public class UsageMetricRecord extends UnifiedRecordValue implements UsageMetricRecordValue {
+
+  private final EnumProperty<IntervalType> intervalTypeProp =
+      new EnumProperty<>("intervalType", IntervalType.class, IntervalType.ROLLING);
```

Comment:
💭  I was wondering if we need the `intervalType` property in the record value. I guess we don't need it now, but if we eventually add additional buckets that are differentiated only by the `Enum`, we will need it.